src/main/java/com/ebay/myriad/configuration/MyriadConfiguration.java

```diff
@@ -145,6 +153,14 @@ public String getFrameworkRole() {
         return frameworkRole;
     }
 
+    public Optional<String> getFrameworkUser() {
+        return Optional.fromNullable(frameworkUser);
+    }
+
+    public Optional<String> getFrameworkSuperUser() {
+        return Optional.fromNullable(frameworkSuperUser);
```

Comment:
You could enforce the default fallback behaviors here instead of at the call-sites.